app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -278,7 +281,11 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
             {this.getStatusIcon(OPEN_APP)}
           </div>
           <div className={styles.ledgerStageText}>
-            <FormattedHTMLMessage id="auth.ledger.navigateToNeoApp" />
+            {chain === 'neo2' ? (
+              <FormattedHTMLMessage id="auth.ledger.navigateToNeoApp" />
+            ) : (
+              <FormattedHTMLMessage id="auth.ledger.navigateToNeo3App" />
```

Comment:
Can we just use the string `'Navigate to the <strong>NEO3 app</strong> on your device'` here - reason being is that without the other translation file updated users will just see `auth.ledger.navigateToNeo3App` when not using English